src/lib/security.js

```diff
@@ -28,3 +29,13 @@ export function allowIframe(): boolean {
 export const protectedExport = (unprotectedExport) =>
   isPayPalDomain() ? unprotectedExport : undefined;
 /* eslint-enable no-confusing-arrow */
+
+// $FlowIssue
+export const localOrStageExport = (unprotectedExport) => {
```

Comment:
Can we rename it to `devEnvOnlyExport` 🤔 ?

Reply:
Much better name :)